packages/ecommerce-ui/components/FormComponents/TextFieldElement.tsx

```diff
@@ -22,6 +22,7 @@ export function TextFieldElement<TFieldValues extends FieldValues>({
   name,
   control,
   defaultValue,
+  castToNumber = true,
```

Comment:
Why would we want to be able to manipulate this value via a prop? Looks to me like this should work out of the box